### PR TITLE
news: give warning about bugtracker downtime

### PR DESCRIPTION
**Pull request by Swiftb0y**

### CODE REVIEW

File: content/news/XXXX-XX-XX-gh-issue-migration-downtime.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+title: Bugtracker temporarily unavailable
+authors: Nikolaus Einhauser
+tags: infrastructure, development, contribute
+comments: yes
+status: draft
+
+
+Hello Everybody.
+
+After much preparation, we're moving from Launchpad to Github issues. We'll
```
Comment (daschuer) [inline content/news/XXXX-XX-XX-gh-issue-migration-downtime.md]:
```suggestion
After thorough preparation, we're moving from Launchpad bugs to GitHub issues.
```

File: content/news/XXXX-XX-XX-gh-issue-migration-downtime.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+publish a more detailed blogpost after the migration is done. For now, We'd
+just like to let everybody know, that we will disable [our bugtracker on
+launchpad](https://bugs.launchpad.net/mixxx) sometime afternoon Monday the 22.
+of August. Any activity on Launchpad from that point on will not be carried over
+to Github. Any already existing link to an issue on Launchpad will still
```
Comment (daschuer) [inline content/news/XXXX-XX-XX-gh-issue-migration-downtime.md]:
```suggestion
to GitHub. Any already existing link to an issue on Launchpad will still
```

File: content/news/XXXX-XX-XX-gh-issue-migration-downtime.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+continue to work, but you will not be able to search for them on Launchpad anymore. Once the
+migration has been completed, the more detailed blogpost will be published and
+you will be able to continue to interact with the newly migrated Issues on [our
```
Comment (daschuer) [inline content/news/XXXX-XX-XX-gh-issue-migration-downtime.md]:
```suggestion
you will be able to interact with the newly migrated issues on [our
```

File: content/news/XXXX-XX-XX-gh-issue-migration-downtime.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+Github repository](https://github.com/mixxxdj/mixxx/issues).
```
Comment (daschuer) [inline content/news/XXXX-XX-XX-gh-issue-migration-downtime.md]:
```suggestion
GitHub repository](https://github.com/mixxxdj/mixxx/issues).
```

File: content/news/XXXX-XX-XX-gh-issue-migration-downtime.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+comments: yes
+status: draft
+
+
+Hello Everybody.
+
+After much preparation, we're moving from Launchpad to Github issues. We'll
+publish a more detailed blogpost after the migration is done. For now, We'd
+just like to let everybody know, that we will disable [our bugtracker on
+launchpad](https://bugs.launchpad.net/mixxx) sometime afternoon Monday the 22.
```
Comment (daschuer) [inline content/news/XXXX-XX-XX-gh-issue-migration-downtime.md]:
```suggestion
launchpad](https://bugs.launchpad.net/mixxx) sometime at Monday the 22nd
```
Afternoon depends on the time zone.

File: content/news/XXXX-XX-XX-gh-issue-migration-downtime.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+of August. Any activity on Launchpad from that point on will not be carried over
+to Github. Any already existing link to an issue on Launchpad will still
+continue to work, but you will not be able to search for them on Launchpad anymore. Once the
+migration has been completed, the more detailed blogpost will be published and
+you will be able to continue to interact with the newly migrated Issues on [our
+Github repository](https://github.com/mixxxdj/mixxx/issues).
+
```
Comment (daschuer) [inline content/news/XXXX-XX-XX-gh-issue-migration-downtime.md:20]:
```suggestion
In the meantime you are welcome to discuss bugs and feature requests at [Zulip](https://mixxx.zulipchat.com).
```

File: content/news/XXXX-XX-XX-gh-issue-migration-downtime.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+authors: Nikolaus Einhauser
+tags: infrastructure, development, contribute
+comments: yes
+status: draft
+
+
+Hello Everybody.
+
+After much preparation, we're moving from Launchpad to Github issues. We'll
+publish a more detailed blogpost after the migration is done. For now, We'd
```
Comment (daschuer) [inline content/news/XXXX-XX-XX-gh-issue-migration-downtime.md]:
```suggestion
For now, We'd
```
The sentence is repeated below.

File: content/news/XXXX-XX-XX-gh-issue-migration-downtime.md [hunk 8]
```diff
@@ -0,0 +1,21 @@
+title: Bugtracker temporarily unavailable
```
Comment (daschuer) [inline content/news/XXXX-XX-XX-gh-issue-migration-downtime.md]:
```suggestion
title: Bug tracker temporarily unavailable
```

File: content/news/XXXX-XX-XX-gh-issue-migration-downtime.md [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+tags: infrastructure, development, contribute
+comments: yes
+status: draft
+
+
+Hello Everybody.
+
+After much preparation, we're moving from Launchpad to Github issues. We'll
+publish a more detailed blogpost after the migration is done. For now, We'd
+just like to let everybody know, that we will disable [our bugtracker on
```
Comment (daschuer) [inline content/news/XXXX-XX-XX-gh-issue-migration-downtime.md]:
```suggestion
just like to let everybody know, that we will disable [our bug tracker on
```

File: content/news/XXXX-XX-XX-gh-issue-migration-downtime.md [hunk 10]
```diff
@@ -0,0 +1,21 @@
...
+launchpad](https://bugs.launchpad.net/mixxx) sometime afternoon Monday the 22.
+of August. Any activity on Launchpad from that point on will not be carried over
+to Github. Any already existing link to an issue on Launchpad will still
+continue to work, but you will not be able to search for them on Launchpad anymore. Once the
+migration has been completed, the more detailed blogpost will be published and
```
Comment (daschuer) [inline content/news/XXXX-XX-XX-gh-issue-migration-downtime.md]:
```suggestion
migration has been completed, the more detailed blog post will be published and
```

File: content/news/XXXX-XX-XX-gh-issue-migration-downtime.md [hunk 11]
```diff
@@ -0,0 +1,21 @@
...
+status: draft
+
+
+Hello Everybody.
+
+After much preparation, we're moving from Launchpad to Github issues. We'll
+publish a more detailed blogpost after the migration is done. For now, We'd
+just like to let everybody know, that we will disable [our bugtracker on
+launchpad](https://bugs.launchpad.net/mixxx) sometime afternoon Monday the 22.
+of August. Any activity on Launchpad from that point on will not be carried over
```
Comment (daschuer) [inline content/news/XXXX-XX-XX-gh-issue-migration-downtime.md]:
```suggestion
of August. Any activity on Launchpad after that point will not be carried over
```

**Review by daschuer — CHANGES_REQUESTED** [review]
Some suggestions and nit-picks. 

Thank you for these efforts.

File: content/news/XXXX-XX-XX-gh-issue-migration-downtime.md (hunk 5 above)
Comment (Swiftb0y) [reply content/news/XXXX-XX-XX-gh-issue-migration-downtime.md]: I could just include the timezone
```suggestion
launchpad](https://bugs.launchpad.net/mixxx) sometime on Monday the 22nd
```

File: content/news/XXXX-XX-XX-gh-issue-migration-downtime.md (hunk 6 above)
Comment (Swiftb0y) [reply content/news/XXXX-XX-XX-gh-issue-migration-downtime.md:20]: Does it really make sense to fragment issue discussion even more?

File: content/news/XXXX-XX-XX-gh-issue-migration-downtime.md (hunk 7 above)
Comment (Swiftb0y) [reply content/news/XXXX-XX-XX-gh-issue-migration-downtime.md]: I'm aware, but I wanted to mention the extra blogpost upfront. Afterwards I need it to refer to the time when its published.

**Comment by Swiftb0y** [comment]
Thanks for the review, though I don't agree with two of your comments.

File: content/news/XXXX-XX-XX-gh-issue-migration-downtime.md (hunk 6 above)
Comment (Holzhaus) [reply content/news/XXXX-XX-XX-gh-issue-migration-downtime.md:20]: I agree. People will just have to wait a little.

File: content/news/XXXX-XX-XX-gh-issue-migration-downtime.md [hunk 12]
```diff
@@ -0,0 +1,21 @@
+title: Bug tracker temporarily unavailable
```
Comment (Holzhaus) [inline content/news/XXXX-XX-XX-gh-issue-migration-downtime.md]: Title case?

File: content/news/XXXX-XX-XX-gh-issue-migration-downtime.md [hunk 13]
```diff
@@ -0,0 +1,21 @@
...
+authors: Nikolaus Einhauser
+tags: infrastructure, development, contribute
+comments: yes
+status: draft
+
+
+Hello Everybody.
+
+After thorough preparation, we're moving from Launchpad bugs to GitHub issues.
```
Comment (Holzhaus) [inline content/news/XXXX-XX-XX-gh-issue-migration-downtime.md]:
```suggestion
After thorough preparation, we're finally moving from Launchpad bugs to GitHub issues.
```

File: content/news/XXXX-XX-XX-gh-issue-migration-downtime.md (hunk 7 above)
Comment (Holzhaus) [reply content/news/XXXX-XX-XX-gh-issue-migration-downtime.md]: I'm any case the sentence is incomplete and should be fixed or removed.

Comment (Swiftb0y) [reply content/news/XXXX-XX-XX-gh-issue-migration-downtime.md]: fixed

File: content/news/XXXX-XX-XX-gh-issue-migration-downtime.md (hunk 5 above)
Comment (daschuer) [reply content/news/XXXX-XX-XX-gh-issue-migration-downtime.md]: Yes that is probably the best.

File: content/news/XXXX-XX-XX-gh-issue-migration-downtime.md (hunk 6 above)
Comment (daschuer) [reply content/news/XXXX-XX-XX-gh-issue-migration-downtime.md:20]: Ok

**Review by daschuer — APPROVED** [review]
Thank you.

**Comment by daschuer** [comment]
@Holzhaus please merge if you are satisfied as well.

**Pull request merged**